src/Ieemdb.Web/Controllers/MovieController.cs

```diff
@@ -85,6 +85,45 @@ public MovieController(ILogger<MovieController> logger, IeemdbDbContext ctx, IPu
       return Ok(x);
     }
 
+    /// <summary>
+    /// This controller gets the list of featured movies.
+    /// </summary>
+    /// <param name="itemsPerPage">Define how many items shall be returned. </param>
+    /// <param name="page">Choose which page of the results shall be returned.</param>
+    /// <returns></returns>
+    [HttpGet("featured")]
+    public async Task<ActionResult<ICollection<MovieDto>>> GetAllFeaturedMovies(int itemsPerPage = 20, int page = 1)
+      {
+      var toSkip = itemsPerPage * (page - 1);
+
+      var moviesQuery = Context.Movies
+        .TagWith("Retrieving all featured movies")
+        .Where(x => x.Featured.Equals(true))
+        .OrderBy(x => x.Id);
+
+      var totalMovies = await moviesQuery.CountAsync();
+
+      if (page > ((totalMovies / itemsPerPage) + 1))
+      {
+        return BadRequest("Page doesn't exist");
+      }
+
+      var pagedMovies = await moviesQuery
+        .Skip(toSkip)
+        .Take(itemsPerPage)
+        .ToListAsync();
+
+      var result = new PagedResult<MovieDto>
+      {
+        Results = pagedMovies.Select(x => Mapper.Map<Movie, MovieDto>(x)).ToList(),
```

Comment:
Stops working on mapping

Reply:
Well of course it does. There is no mapper configured for Movie to MovieDto. 

https://github.com/esentis/ieemdb-adopse-2021/blob/fbe976238086f6e39cbb39682b95f486259e6eec/src/Ieemdb.Web/Helpers/MappingConfiguration.cs#L12